packages/sn-filter-pane/src/ext/property-panel/data/data-panel/presentation/index.ts

```diff
@@ -3,9 +3,12 @@ import { IGenericListPropertiesMissing } from '../../../../../../../../types/glo
 import textAlignItems from '../../text-align-items';
 import { IEnv } from '../../../../../types/types';
 import frequencies from '../../../constants';
+import updateFrequencyMax from '../update-frequency-max';
```

Comment:
It seems like this function actually does more than changing the frequency max, it also changes the qFrequencyMode property, so the naming is a little bit confusing then to be strict.